flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/operations/OperationTreeBuilder.scala

```diff
@@ -160,11 +159,61 @@ class OperationTreeBuilder(private val tableEnv: TableEnvImpl) {
     aggregateOperationFactory.createAggregate(resolvedGroupings, resolvedAggregates, child)
   }
 
+  def rowBasedAggregate(
```

Comment:
I think we can `rowBasedAggregate` -> `aggregate` and add some comments to illustrate the difference between the two aggregate methods, What do you think?